packages/backend/package.json

```diff
@@ -30,7 +30,8 @@
     "imgix-core-js": "2.3.2",
     "node-fetch": "2.6.1",
     "graphql-request": "3.2.0",
-    "sourcecred": "0.7.6"
+    "sourcecred": "0.7.6",
+    "sourcecred-publish-test": "0.7.20"
```

Comment:
¿This package is a pre-release version of the main SourceCred package?

¿Have you considered doing a `@next` version of the package, like Ceramic does, which lets you avoid installing two separate packages?